pkg/tfbridge/schema.go

```diff
@@ -293,6 +293,8 @@ type conversionContext struct {
 	ApplyDefaults         bool
 	ApplyTFDefaults       bool
 	Assets                AssetTable
+	// UseTFSetTypes will output TF Set types when converting sets.
```

Comment:
This doc comment could be a lot better and give an example for false and true values. Could save someone time looking this up later on.